change/@ni-nimble-blazor-956c2fbb-7d8b-4e19-9c09-d5996fabcc9c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Can make this type none, dependentChangeType node since there are no lib changes